Базовая стилизация всех страниц

Deniso4kin opened the pull request:
1) переименовал две картинки в папке images.
2) создал паку styles/ и и подключил к странице стилевой файл styles.css
3) сделал базовую стилизацию всех страниц

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2252213/tasks/8)

Deniso4kin committed:
переименовал две картинки

Deniso4kin committed:
Базовая стилизация всех страниц

Deniso4kin committed:
Обновлённый catalog.html

ilanabatavraam commented on catalog.html:
```diff
@@ -4,125 +4,139 @@
   <head>
     <meta charset="utf-8">
     <title>Каталог</title>
+    <link rel="stylesheet" href="styles/styles.css">
```
В head обоих страниц необходимо добавить мета-тег:

```
<meta name="viewport" content="width=device-width, initial-scale=1" />

```

ilanabatavraam commented on index.html:
```diff
@@ -3,12 +3,13 @@
 
   <head>
```
Мы и так находимся на главной, ссылку на саму себя стоит убрать (оставить href пустым)
https://skr.sh/sIblnM9aBKA

https://evo.business/ssylki-stranicy-na-samu-sebya/

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
 
 body {
```
Код необходимо отфарматировать во всем css файле.

В VS Code, можно форматировать код автоматически:
https://codengineering.ru/q/how-do-you-format-code-in-visual-studio-code-vscode-27090.
Также, можно пользоваться онлайн-сервисами для форматирования кода:
https://webformatter.com/

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+  height: 485px;
+}
+
+.divider{
```
Я бы этот тег из разметки удалила вообще. 
Данную сущность лучше реализовать псевдоэлементом.

https://doka.guide/css/before/

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+
+.subscribe-catalog {
+  width: 1200px;
+  height: 414.86px;
```
Не нужно фиксировать высоту. Этот стиль необходимо удалить.
Позднее вид блока скорректируется отступами

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+  background-repeat: no-repeat;
+  text-align: center;
+  width: 1200px;
+  height: 485px;
```
Высоту лучше заменить на минимальную высоту

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+  background-color: #A4766B;
+  background-image: url("../images/subscribe.jpg");
+  width: 1200px;
+  height: 414.86px;
```
Не нужно фиксировать высоту. Этот стиль необходимо удалить.
Позднее вид блока скорректируется отступами

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+  background-image: url("../images/catalog-filter.jpg");
+  background-repeat: no-repeat;
+  width: 1200px;
+  height: 412px;
```
Не нужно фиксировать высоту. Этот стиль необходимо удалить.
Позднее вид блока скорректируется отступами

ilanabatavraam commented on styles/styles.css:
```diff
@@ -16,4 +16,353 @@
+
+.pagination-link {
+  color: #ffffff;
+  background-color: #82B3D3;;
```
Лишний знак необходимо удалить

Deniso4kin committed:
В head обеих страниц  добавил мета-тег

В head обеих страниц  добавил мета-тег: <meta name="viewport" content="width=device-width, initial-scale=1" />

Deniso4kin committed:
Убрал из шапки ссылки страниц на самих себя

Deniso4kin committed:
Отфарматировал код в style.css

Deniso4kin committed:
удалил divider из разметки и style.css

Deniso4kin committed:
Убрал высоты у блоков

ilanabatavraam reviewed (commented):
Задание принимаю 🚀
